docs/extend/plugins_volume.md

```diff
@@ -164,3 +164,52 @@ this point.
 
 Respond with a string error if an error occurred.
 
+
+### /VolumeDriver.Get
+
+**Request**:
+```json
+{
+    "Name": "volume_name"
+}
+```
+
+Get the volume info.
+
+
+**Response**:
+```json
+{
+  "Name": "volume_name",
+  "Mountpoint": "/path/to/directory/on/host",
+  "Err": ""
+}
+```
```

Comment:
Hi @cpuguy83! This 'Get' response is not correct. The code expects the response to be inside a "Volume" hash, like:

````
{
    "Volume": {
        "Name": "volume_name",
        "Mountpoint": "/path/to/directory/on/host",
    },
    "Err": ""
}
````

in volume/drivers/proxy.go:
````
      req.Name = name
      if err = pp.Call("VolumeDriver.Get", req, &ret); err != nil {                                                                
          return                                                                                                                   
      }                                                                                                                            
  
      volume = ret.Volume    
````

*Possibly* its more consistent to have "Name" and "Mountpoint" at the top-level, like the rest of the API, and have the response in the proxy parsed into a Volume hash.

Easier to change the code or the docs? :-)